package.json

```diff
@@ -7,7 +7,8 @@
     "start": "craco start",
     "build": "craco build",
     "test": "craco test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint '**/*.ts' '**/*.tsx'"
```

Comment:
Also, add a script called `fix` which can fix the basic eslint errors.

Reply:
Already added in the later PRs.